### PR TITLE
dev-python/python-caja: bump version to 1.22.1

### PR DESCRIPTION
**Pull request by oz123**
Also added python3 support

Bug: https://bugs.gentoo.org/702062
Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: dev-python/python-caja/python-caja-1.22.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MATE_LA_PUNT="yes"
+PYTHON_COMPAT=( python2_7 python3_6 python3_7 )
```
Comment (flappyports) [inline dev-python/python-caja/python-caja-1.22.1.ebuild]: Please drop python2_7 support

Comment (NP-Hardass) [reply dev-python/python-caja/python-caja-1.22.1.ebuild]: As several extensions don't support py3 yet, I'm inclined not to drop it yet.

Comment (oz123) [reply dev-python/python-caja/python-caja-1.22.1.ebuild]: Also, upstream didn't remove it. The next version 1.23 removed python 2.
@NP-Hardass I have prepared updated ebuilds in my overlay. Can you merge this so I can add more PRs with the next ebuilds?

**Comment by juippis** [comment]
Looks like this package doesn't exist in ::gentoo anymore.

**Pull request closed without merging**